addons/mission/functions/fnc_reinforcements.sqf

```diff
@@ -5,7 +5,7 @@
  * Call from initServer.sqf
  *
  * Arguments:
- * 0: Group <GROUP>
+ * 0: Group <GROUP> (default: grpNull)
```

Comment:
There is no default, because next argument is required... Only useful if you pass `nil` directly.

Reply:
It's for someone doing something like this:
`[R_4, false] call MFUNC(reinforcements`

If `R_4` doesn't exist, it'll be `grpNull`.

Reply:
But this header is still wrong. It can't list a default value like that.